### PR TITLE
Etapa1

### PR DESCRIPTION
**Pull request by DutuMaria**

### CODE REVIEW

File: src/entity/user/Customer.java [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+    public void viewHotelServices(){
+        Hotel hotel = Hotel.getHotelInstance();
+        System.out.println(hotel.getHotelServices());
+    }
+
+    public void viewAvailableRooms(){
+        Hotel hotel = Hotel.getHotelInstance();
+        System.out.println(hotel.getRoomList().stream().filter(x -> x.getRoomStatus() == RoomStatus.AVAILABLE).toList());
+    }
```
Comment (alina-puscasu) [inline src/entity/user/Customer.java]: line 95 does not compile on my laptop - does it work on yours? 
fix: System.out.println(hotel.getRoomList().stream().filter(x -> x.getRoomStatus() == RoomStatus.AVAILABLE).collect(Collectors.toList())); - you need to collect the elements of the stream generated by the filter method in a List using Collectors

Comment (DutuMaria) [reply src/entity/user/Customer.java]: La mine compileaza tot, dar voi rezolva. Multumesc!

File: src/entity/user/Admin.java [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+                room.setRoomType(roomType);
+            }
+        }
+    }
+
+    public void deleteRoom(int roomNumber){
+        // TODO: verificare daca aceasta camera are booking-uri asociate => in viitor => schimbi respectivul booking, daca este posibil
+        //                                                               => in prezent => nu poti sterge camera
+        hotel.getRoomList().removeIf(x-> x.getRoomNumber() == roomNumber);
+    }
```
Comment (alina-puscasu) [inline src/entity/user/Admin.java]: all these methods should be in the service for Admin - AdminService, in entities we have just fields, setters, getters, constructors, toString, equals, hashCode

File: src/entity/user/Customer.java [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+                    System.out.println("Password has been successfully changed!");
+                } else {
+                    System.out.println("New password cannot be the same as your old password.");
+                }
+            }
+        } else {
+            System.out.println("Incorrect password! You can't change your password.");
+        }
+    }
+
```
Comment (alina-puscasu) [inline src/entity/user/Customer.java]: same here - logic should be kept in services not in entities

**Comment by alina-puscasu** [comment]
poti face merge in main cand doresti :)

**Pull request merged**